charts/k8ssandra/Chart.yaml.orig

```diff
@@ -0,0 +1,44 @@
+apiVersion: v2
+name: k8ssandra
+description: |
+  Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
+type: application
+version: 1.3.0-SNAPSHOT
+
+dependencies:
+  - name: cass-operator
+    version: 0.29.4
+    repository: file://../cass-operator
+    condition: cass-operator.enabled
+
+  - name: reaper-operator
+<<<<<<< HEAD
```

Comment:
We've got another stowaway `.orig` file here. Let's remove it like the last one.

Reply:
🤦